tests/README.md

```diff
@@ -2,25 +2,20 @@
+* Test cases to check existence of in-tree and OOT tree drivers
 * Any test cases for SELinux and host OS security
-* go-based tests cases for devices on all nodes
-* Dependency- rely on driver teams for any other testing result 
-* Future plan to include OOT driver test cases with KMMO CI/CD pipeline- sanity testing.
-* Currently execute as a job. Plan to find an efficient mechanism to deploy and automate on all nodes.
-
+* go-based tests cases to check for devices on all nodes
+  
 #### L2 Second Layer: Device plugin resource provisioning  
-This layer will be executed after the Intel Device Plugins Operator installation and Custom Resource creation. It includes 
+This layer will be executed after the Intel Device Plugins Operator installation and [device plugins](https://github.com/vbedida79/intel-technology-enabling-for-openshift/blob/main/device_plugins/README.md) creation. Refer to [readme](https://github.com/intel/intel-technology-enabling-for-openshift/blob/main/tests/l2/README.md) It includes: 
 * Pod's resource allocation and scheduling 
 * Simple workloads 
 * Boundary testing for the resources
-* Integration and stress testing 
-* Future plan to include any OOT driver test cases in KMMO CI/CD pipeline- canary testing 
-* Any failure analysis needed during automation, in the future. 
+* Future plan for any failure analysis needed during automation. 
 
 #### L3 Third Layer: E2E solution
-This layer consists of the reference workload's/fourth layer's prerequisites and dependencies- after the cluster, operators and resources are accurately deployed. It is custom for each Intel Hero feature's final use case. For example, dGPU smart city workload would have RHODS (Red Hat OpenShift Data Science) and OpenVINO operator as prerequisites included here.
+This layer consists of the reference workload's/fourth layer's prerequisites and dependencies- after the cluster, operators and resources are accurately deployed. It is custom for each device plugin and includes workloads using device plugins resources. Refer to [readme](https://github.com/intel/intel-technology-enabling-for-openshift/blob/main/tests/l3/README.md). For example, GPU would have [RHODS](https://catalog.redhat.com/software/container-stacks/detail/63b85b573112fe5a95ee9a3a) (Red Hat OpenShift Data Science) and [OpenVINO operator](https://catalog.redhat.com/software/operators/detail/60649a56209af65d24b7ca9e) based workload.
```

Comment:
L3 test cases can be executed after the related e2e solution has been deployed in [day 2](../README.md#day-2---ocp-platform-is-installed-and-ready-to-begin-providing-services).  Please refer to [L3 test cases](l3/README.md) for detail.